docs/application/dotnet/guides/app-management/application-launcher.md

```diff
@@ -0,0 +1,304 @@
+# The Application Launcher
```

Comment:
Please avoid articles in headings, also my previous comment was to change this heading to title case it is not fixed.
Do you want to retain both A and L caps?

If not suggest to change to
# Application launcher